src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,39 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.EmptyDescriptionException;
+import duke.task.Task;
+
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FindCommand extends Command {
+    private String keyword;
+
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws IOException, EmptyDescriptionException, ParseException {
+        List<Task> relevantList = new ArrayList<>();
+        for(int i=0;i<tasks.listSize(); i++){
```

Comment:
Coding style: consider adding spaces in your parenthesis arguments.
`for (int i = 0; i < tasks.listSize(); i++) {`

Reply:
Make use of IDE to refactor your code to comply to the coding style